lib/src/entrypoint.dart

```diff
@@ -370,8 +370,8 @@ class Entrypoint {
     var executablesExist = executables.every((executable) {
       var snapshotPath = p.join(
           _snapshotPath, packageName, "${p.basename(executable)}.snapshot");
-      if (!fileExists(snapshotPath)) return false;
-      if (isDart2 && !fileExists("$snapshotPath.dart2")) return false;
+      return fileExists(snapshotPath) &&
+          (!isDart2 || fileExists("$snapshotPath.dart2"));
```

Comment:
nit: just add an explicit return at the end, this logic is much harder to parse

Reply:
haha I thought the declarative version is easier to understand than tracing the imperative flow, but I can see how the more complex boolean statement could change the balance on that.